### PR TITLE
[TS] LPS-162975

### PR DESCRIPTION
**Pull request by crodriguezy**
https://issues.liferay.com/browse/LPS-162975

@marcoscv-work this is the minimal fix based on [PTR](https://issues.liferay.com/browse/PTR-3308), can you take a look?

The changes have to be backported to 7.2.x because this is a need in that version.

cc/ @jcampoy

### CODE REVIEW

**Comment by jcampoy** [comment]
Hi @marcoscv-work ! could you take a look to this PR?

Thanks!

**Comment by crodriguezy** [comment]
Hi @liferay-frontend , could you please review this PR?
Thank you.

**Comment by dsanz** [comment]
ci:test:relevant

File: portal-web/docroot/html/taglib/ui/search_iterator/deprecated/list.jsp [hunk 1]
```diff
@@ -145,12 +148,22 @@ if (iteratorURL != null) {
 						<c:if test="<%= (rowChecker != null) && (i == 0) %>">
 							width="1%"
 						</c:if>
+
+						<c:if test="<%= (orderKey != null) && orderCurrentHeader %>">
+							aria-live="assertive" aria-sort="<%= LanguageUtil.get(resourceBundle, sortedActionMessageKey) %>"
```
Comment (dsanz) [inline portal-web/docroot/html/taglib/ui/search_iterator/deprecated/list.jsp:153]: @marcoscv-work @drakonux  There's not many places in the product with `aria-live="assertive"`. I've only seen them when reporting errors. 
Do we have any recommendations or guidance about using `assertive` vs `polite`?

**Comment by eltonccdantas** [comment]
I'll take a look at it

**Comment by eltonccdantas** [comment]
Hi @crodriguezy , I've analyzed the errors and they look unrelated to your PR according to the [comparison.](https://testray.liferay.com/home/-/testray/runs/compare?compare=true&view=details&statusA=2&testrayRunIdB=2052638051&testrayRunIdA=2054443870&statusB=3) You may manually forward if no other dev review is needed. 

CC @mateusralv @john-co

**Comment by crodriguezy** [comment]
Ok, thank you @eltonccdantas . I'll forward it

**Comment by crodriguezy** [comment]
Sent to https://github.com/brianchandotcom/liferay-portal/pull/126567

**Pull request closed without merging**